dd-sdk-android-internal/src/main/java/com/datadog/android/internal/telemetry/InternalTelemetryEvent.kt

```diff
@@ -6,7 +6,11 @@
 
 package com.datadog.android.internal.telemetry
 
+import com.datadog.android.internal.utils.loggableStackTrace
+import com.datadog.android.lint.InternalApi
+
 @Suppress("UndocumentedPublicClass", "UndocumentedPublicFunction", "UndocumentedPublicProperty")
+@InternalApi
```

Comment:
> [!Note]
> Why function in internal module needs to be marked as `@InternalApi`?

Reply:
In case somehow this is exposed through some module using `api`. I think it is safer to mark all these methods and classes with `@InternalApi`.

Reply:
That's also true, let's remove these then.